### DISCUSSION TITLE
edit existing shape in Qfield

### DISCUSSION THREAD

**Discussion by gandlar** [opengisch/QField#3598] [Q&A]
Is it possible to edit existing shape in the Qfield app? For example, I plan to make project with polygons of buildings for which I want to enter attributes on the field.
Thanks

**Comment by suricactus** [comment]
It is possible to edit geometry in QField, just make sure you are in editing mode enabled from the left drawer. Read more here: https://docs.qfield.org/how-to/digitize/
